### ISSUE TITLE
Number of time frames in wrong position in interfile header

### ISSUE THREAD

**Issue by ALEXJAZZ008008** [UCL/STIR#320]
The line: "number of time frames :=" in Interfile headers is above the line: "!PET data type :=" rather than above the block: "first pixel offset (mm) [1] := \nfirst pixel offset (mm) [2] := \nfirst pixel offset (mm) [3] := "

This is causing issues when trying to open the Interfile images to view or convert to Nifty.

**Comment by rijobro** [comment]
Why does this cause a problem? (haven't look at the code...)
The number of time frames and the offset seem to be unrelated in my mind.

**Comment by ALEXJAZZ008008** [comment]
I'm not certain why it causes a problem, when I opened an Interfile header in Amide before it was entirely blank, I asked Elise about it and she told me the issue was caused by this ordering.
If I manually move the line in my Interfile header it causes Amide to be able to open the file as expected.
I didn't want to manually move the line for every one of my Interfile files and I don't know if it affects converting Interfile to Nifty.

I'm running with my edited branch where this change has been made and it seems fine to me, although I haven't had long to test it.
